packages/web/src/components/Error/ErrorPopup.tsx

```diff
@@ -197,14 +289,18 @@ export function ErrorPopupDisconnected({ globalError, algorithmErrors, errorDism
           <ErrorContent error={error} />
           <section className="mt-3">
             <div>
-              {t('If you think it might be a bug in {{appName}}, please create a new issue at:', {
+              {t('If you think it is a bug in {{appName}}, report it at', {
                 appName: PROJECT_NAME,
               })}
             </div>
             <div>
               <LinkExternal href={URL_GITHUB_ISSUES}>{URL_GITHUB_ISSUES_FRIENDLY}</LinkExternal>
             </div>
-            <div>{t('The developers will be happy to investigate this problem.')}</div>
+            <div>
+              {t(
+                'so that developers could investigate this problem. Please provide as much details as possible about your input data, operating system, browser version, computer configuration. include other details you deem useful for the diagnostics. Share the example sequence data that allows to reproduce the problem, if possible.',
```

Comment:
```suggestion
                'so that developers can investigate this problem. Please provide as much details as possible about your input data, operating system, browser version and computer configuration. Include other details you deem useful for diagnostics. Share the example sequence data that allows to reproduce the problem, if possible.',
```